### PR TITLE
V5 Docs - `getPublicSuffix`

### PR DESCRIPTION
**Pull request by colincasey**
Rendered pages:
- https://github.com/salesforce/tough-cookie/blob/v5-docs-getpublicsuffix/api/docs/tough-cookie.getpublicsuffix.md
- https://github.com/salesforce/tough-cookie/blob/v5-docs-getpublicsuffix/api/docs/tough-cookie.getpublicsuffixoptions.md

---
Main PR:
- #384

### CODE REVIEW

File: api/docs/tough-cookie.getpublicsuffixoptions.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+## GetPublicSuffixOptions type
+
+Options for configuring how [getPublicSuffix()](./tough-cookie.getpublicsuffix.md) behaves.
+
+**Signature:**
+
+```typescript
+export type GetPublicSuffixOptions = {
+    allowSpecialUseDomain?: boolean | undefined;
+    ignoreError?: boolean | undefined;
```
Comment (wjhsf) [inline api/docs/tough-cookie.getpublicsuffixoptions.md:14]: Can we get the properties annotated, as well?

Comment (colincasey) [reply api/docs/tough-cookie.getpublicsuffixoptions.md:14]: yeah, i also noticed that individual properties didn't show up in the docs after the fact.  but i may have forgot to regenerate these pages after annotating the type.

Comment (colincasey) [reply api/docs/tough-cookie.getpublicsuffixoptions.md]: @wjhsf turns out that the doc tooling won't populate the property information when using a `type` vs. an `interface` so I've switched `export type GetPublicSuffixOptions` to `export interface GetPublicSuffixOptions`.

**Review by wjhsf — APPROVED** [review]
What is the "Modifiers" column on the [type def page](https://github.com/salesforce/tough-cookie/blob/v5-docs-getpublicsuffix/api/docs/tough-cookie.getpublicsuffixoptions.md)?

**Comment by colincasey** [comment]
@wjhsf it contains extra information like `readonly` or `static`. E.g.; https://github.com/salesforce/tough-cookie/blob/v5-docs-cookiejar/api/docs/tough-cookie.cookiejar.md

**Pull request merged**